server/core/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -364,7 +363,12 @@ object Parse {
         startElement.name.map(_.value).getOrElse("???"),
         /* If sourceReference > 0 the contents of the source must be retrieved through
          * the SourceRequest (even if a path is specified). */
-        new Types.Source(schemaPath.toString, 0),
+        Try(Paths.get(URI.create(startElement.schemaLocation.uriString)).toString())
```

Comment:
So far in practice the `schemaLocation.uriString` has proved valid for jar-based schemas. They look like `jar:file:/x/y/test.jar!/foo/bar.xsd` and parse correctly.

If anything might change in the schema location, it would be to use a `URI` instead of a `String`.